### PR TITLE
Enable rt.sh dependency relations

### PR DESCRIPTION
**Pull request by MinsukJi-NOAA**
Issue #103 
*Used to create baselines in one step without manual copying of files
*Used for restart rt to get initial data from work directory
*Merge cpld_fv3_mom6_cice_atm_flux_restart_run.IN into cpld_fv3_mom6_cice_atm_flux_run.IN

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
So looking through the tests, it seems like for now we're not implementing the restart test to use the intermediate restarts. Is the plan is first to implement the dependency function and then later move to using the intermediate restarts? At that point, the 3d run tests could be deleted.

**Comment by MinsukJi-NOAA** [comment]
That's right. The restart run in this PR stays mostly the same as before. The only change is that it is grabbing the restart data from a current run (not the baseline data) whose name is specified as a dependency in rt.conf.

**Review by DeniseWorthen — APPROVED** [review]
I understand the changes in test/fv3_conf, tests/tests and rt.conf and they look right.

**Comment by DeniseWorthen** [comment]
Is there any reason why the dependency would not work with the 35d optional tests that Bin added?

**Comment by MinsukJi-NOAA** [comment]
I did not look into the 35d tests. Looks like we just need to add the dependencies in the 6th column of rt_35d_*.conf files.

File: tests/fv3_conf/fv3_slurm.IN [hunk 1]
```diff
@@ -3,40 +3,36 @@
 #SBATCH -o out
 #SBATCH --account=@[ACCNR]
 #SBATCH --qos=@[QUEUE]
-#SBATCH --partition=@[PARTITION]
```
Comment (junwang-noaa) [inline tests/fv3_conf/fv3_slurm.IN:6]: This is a temporary file created from fv3_slurm.IN_MACHINE, maybe we can delete it in next commit.

Comment (MinsukJi-NOAA) [reply tests/fv3_conf/fv3_slurm.IN:6]: removed the temporary fv3_slurm.IN

**Comment by DeniseWorthen** [comment]
I ran into a problem trying to run the 35d test w/ the dependency on orion. I got
cp: cannot stat '/work/noaa/stmp/jminsuk/RT/FV3-MOM6-CICE5/develop-20200530/WW3_input_data/ww3_multi_35d.inp': No such file or directory

@binli2337 Is there additional input required in the baseline for the 35d test

**Comment by MinsukJi-NOAA** [comment]
Which test case was that?

**Comment by DeniseWorthen** [comment]
I tried the rt_35d_cmeps.conf after you added the dependency columns. I merged your rt-dependency branch  to my develop branch.

**Comment by MinsukJi-NOAA** [comment]
I am running rt_35d_cmeps.conf on Orion right now.

**Comment by MinsukJi-NOAA** [comment]
Dependency implementations in this PR actually does not affect the 35 day runs, because we are neither generating the baselines nor testing the restart capability. Nevertheless, the 35d cmeps cold run were completed successfully. Warm runs did not complete because of the issue here: https://github.com/ufs-community/ufs-s2s-model/pull/111#discussion_r445524864

**Comment by MinsukJi-NOAA** [comment]
This PR is ready for merge. Other 35-day run issues should be addressed separately.

**Review by junwang-noaa — APPROVED** [review]

**Pull request merged**